src/AuthorizationServerFactory.php

```diff
@@ -31,10 +31,11 @@ class AuthorizationServerFactory
 
     /**
      * @param ContainerInterface $container
+     * @param string $requestedName
      *
      * @return AuthorizationServer
      */
-    public function __invoke(ContainerInterface $container) : AuthorizationServer
+    public function __invoke(ContainerInterface $container, $requestedName) : AuthorizationServer
```

Comment:
```suggestion
    public function __invoke(ContainerInterface $container, $requestedName = AuthorizationServer::class) : AuthorizationServer
```

If we make the argument optional, with a sane default, we keep backwards compatibility.

Reply:
It's absolutely OK for me :)